src/swim/swim.c

```diff
@@ -116,8 +118,36 @@ swim_updates_send(struct swim_context *ctx, swim_id_t id, swim_id_t to)
 out_unlock:
 	swim_ctx_unlock(ctx);
 
-	if (rc == 0)
-		rc = ctx->sc_ops->send_message(ctx, to, upds, i);
+	if (rc == 0) {
+		nupds = i;
+		if (d_log_check(DLOG_INFO | D_LOGFAC)) {
```

Comment:
Also, please use the new macro for this check

if (D_LOG_ENABLED(DLOG_INFO)) {
}

This will use cached D_LOGFAC + DLOG_INFO mask and avoid multiple calls to d_log_check

Reply:
thanks.